assets/src/scripts/web-services/statistics-data.js

```diff
@@ -6,6 +6,16 @@ import {parseRDB} from 'ui/utils';
 
 const SERVICE_ROOT = config.SERVICE_ROOT || 'https://waterservices.usgs.gov/nwis';
 
+/**
+ *  Formats a URL for the purpose of downloading median statistics in RDB format
+ *  @param  {String}  siteno - A code that uniquely identifies a monitoring location
+ *  @param  {String}  parameterCode - USGS five digit parameter code
+ *  @return {String} The URL used to contact waterservices.usgs.gov
+ */
+export const getServiceURLStatistics = function({siteno, parameterCode}) {
```

Comment:
As with the groundwater levels, use this function in the fetch function to form the URL.

Reply:
Oh, duh. Will do.